### PR TITLE
Consolidate RHEL/CentOS user guide into a single document.

### PR DESCRIPTION
**Pull request by dwsteele**
The user guide was split primarily to provide documentation for the stop-auto option in PostgreSQL <= 9.5. Now that 9.5 is EOL there does not seem to be a good reason to generate an extra user guide. The stop-auto function is still documented in the reference.

Leave the stop-auto documentation in the user guide in case we want to manually generate documentation for older versions.

Also rename centos to rhel for most identifiers since that is the core platform we are building for, similar to how we label 'debian' builds even though we generally use Ubuntu. With CentOS set to become an upstream for RHEL later this year, we'll likely need to pick a new test distribution, perhaps Rocky Linux if that gets off the ground.

### CODE REVIEW

File: doc/xml/documentation.xml [hunk 1]
```diff
@@ -72,16 +73,16 @@ sudo usermod -aG docker testdoc
 ./doc.pl --out=html --include=user-guide --no-cache --var=os-type=debian --var=os-image=ubuntu:16.04 --var=package=test/package/pgbackrest_2.08-0_amd64.deb
         </code-block>
 
-        <p>CentOS/RHEL 7:</p>
+        <p>RHEL/CentOS 7:</p>
 
         <code-block type="bash">
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos7 --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos --var=os-image=centos:7  --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
```
Comment (cmwshang) [inline doc/xml/documentation.xml:79]: os-type should be rhel?

File: doc/xml/documentation.xml [hunk 2]
```diff
@@ -72,16 +73,16 @@ sudo usermod -aG docker testdoc
...
         </code-block>
 
         <p>CentOS/RHEL 8:</p>
 
         <code-block type="bash">
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos8 --var=package=test/package/pgbackrest-2.08-1.el8.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos --var=os-image=centos:8 --var=package=test/package/pgbackrest-2.08-1.el8.x86_64.rpm
```
Comment (cmwshang) [inline doc/xml/documentation.xml:85]: os-type should be rhel?

File: doc/xml/documentation.xml (hunk 1 above)
Comment (dwsteele) [reply doc/xml/documentation.xml]: Yes, fixed in 201b52c.

File: doc/xml/documentation.xml (hunk 2 above)
Comment (dwsteele) [reply doc/xml/documentation.xml]: Yes, fixed in 201b52c.

File: doc/README.md [hunk 3]
```diff
@@ -55,13 +56,13 @@ Ubuntu 16.04:
 ```bash
 ./doc.pl --out=html --include=user-guide --no-cache --var=os-type=debian --var=os-image=ubuntu:16.04 --var=package=test/package/pgbackrest_2.08-0_amd64.deb
 ```
-CentOS/RHEL 7:
+RHEL/CentOS 7:
 ```bash
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos7 --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=rhel --var=os-image=centos:7  --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
 ```
 CentOS/RHEL 8:
```
Comment (cmwshang) [inline doc/README.md:63]: Shouldn't `CentOS/RHEL 8:` have been changed to `RHEL/CentOS 8:`?

File: doc/xml/contributing.xml [hunk 4]
```diff
@@ -283,7 +283,7 @@
...
 
-        <p>If using a RHEL system, the CPAN XML parser is required for running <file>test.pl</file> and <file>doc.pl</file>. Instructions for installing Docker and the XML parse can be found in the <file>README.md</file> file of the <backrest/> <link url="{[github-url-master]}/doc">doc</link> directory in the section <quote>The following is a sample CentOS/RHEL 7 configuration that can be used for building the documentation</quote>. NOTE that the <code>Install latex (for building PDF)</code> is not required since testing of the docs need only be run for HTML output.</p>
+        <p>If using a RHEL-based system, the CPAN XML parser is required for running <file>test.pl</file> and <file>doc.pl</file>. Instructions for installing Docker and the XML parse can be found in the <file>README.md</file> file of the <backrest/> <link url="{[github-url-master]}/doc">doc</link> directory in the section <quote>The following is a sample RHEL/CentOS 7 configuration that can be used for building the documentation</quote>. NOTE that the <code>Install latex (for building PDF)</code> is not required since testing of the docs need only be run for HTML output.</p>
```
Comment (cmwshang) [inline doc/xml/contributing.xml:286]: Doesn't look like the contributing.md was generated for this PR....

File: doc/xml/documentation.xml [hunk 5]
```diff
@@ -72,16 +73,16 @@ sudo usermod -aG docker testdoc
...
 
-        <p>CentOS/RHEL 7:</p>
+        <p>RHEL/CentOS 7:</p>
 
         <code-block type="bash">
-./doc.pl --out=html --include=user-guide --no-cache --var=os-type=centos7 --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
+./doc.pl --out=html --include=user-guide --no-cache --var=os-type=rhel --var=os-image=centos:7  --var=package=test/package/pgbackrest-2.08-1.el7.x86_64.rpm
         </code-block>
 
         <p>CentOS/RHEL 8:</p>
```
Comment (cmwshang) [inline doc/xml/documentation.xml:82]: Ah, this is why the doc/README.md is still showing the CentOS/RHEL vs RHEL/CentOS....

**Review by cmwshang — COMMENTED** [review]
Review complete with some questions.

File: doc/README.md (hunk 3 above)
Comment (dwsteele) [reply doc/README.md:63]: Fixed in 365a558c089.

File: doc/xml/documentation.xml (hunk 5 above)
Comment (dwsteele) [reply doc/xml/documentation.xml:82]: Also fixed in 365a558c089.

File: doc/xml/contributing.xml (hunk 4 above)
Comment (dwsteele) [reply doc/xml/contributing.xml:286]: Reverted in fcff1775a2eff3 since this edit was already made in 68faf1482a42ea14.

File: doc/README.md (hunk 3 above)
Comment (dwsteele) [reply doc/README.md]: And again in 1eb931b8ec99.

File: doc/xml/contributing.xml (hunk 4 above)
Comment (dwsteele) [reply doc/xml/contributing.xml:286]: Undid revert since some of it was still needed in 1eb931b8ec99.

File: doc/xml/documentation.xml (hunk 5 above)
Comment (dwsteele) [reply doc/xml/documentation.xml]: And again in 1eb931b8ec99.

**Review by cmwshang — APPROVED** [review]

**Pull request merged**